bhclub.js

```diff
@@ -22,13 +22,36 @@ $(document).on('click', '.nav-link', function() {
     $('nav').removeClass('show-nav');
     $('#toggle').removeClass('fa-times');
     $('#toggle').addClass('fa-bars');
-    $('main').removeClass('main-pointer');
+    $('.mobile-grayout').removeClass("show-gray");
 });
 
 // Closing the navbar when outside of the nav is clicked
-$(document).on('click', 'main', function() {
+$(document).on('click', '.mobile-grayout', function() {
     $('nav').removeClass('show-nav');
     $('#toggle').removeClass('fa-times');
     $('#toggle').addClass('fa-bars');
-    $('main').removeClass('main-pointer');
+    $('.mobile-grayout').removeClass("show-gray");
 });
+
+// FAQ Cards hide/show
+let card = document.getElementsByClassName("card");
+for (let i = 0; i < card.length; i++) {
+    let accordion = card[i].getElementsByClassName("card-header")[0];
+    // Click should only work on accordion-header of each card
+    accordion.addEventListener("click", function() {
+        card[i].classList.toggle("card-open");
+
+        let panel = card[i].getElementsByClassName("card-body")[0];
+        let fa = this.getElementsByTagName("i")[0];
+
+        // Toggle panel and plus/minus on click of header
+        if ($(card[i]).hasClass("card-open")) {
+            $(panel).slideDown(100);
```

Comment:
These are snappy! But the hover transitions for the colors should also be updated to the same `0.1s` duration, or closer: `1s` hover on the color change seems reeeeeeally slow to me

Reply:
0.1s is too quick imo; trying 0.5s in the upcoming fixes